MSAL/test/unit/MSALPublicClientApplicationTests.m

```diff
@@ -3468,6 +3468,70 @@ - (void)testSignoutWithAccount_whenNonNilAccount_andWipeAccountTrue_andBrokerDis
     [self waitForExpectations:@[expectation] timeout:1];
 }
 
+- (void)testSignoutWithAccount_whenNonNilAccount_andWipeAllAccountsTrue_andBrokerDisabled_shouldWipeAllAccounts
```

Comment:
Would be also good to add a few tests that check wiping for additional partner locations if possible

Reply:
Added additional unit test.